.gitignore

```diff
@@ -207,17 +207,7 @@ $RECYCLE.BIN/
 *.out
 *.app
 
-
-### https://raw.github.com/github/gitignore/218a941be92679ce67d0484547e3e142b2f5f6f0/Global/VisualStudioCode.gitignore
```

Comment:
We want to share these files with the project members. Could you add it?

Reply:
I wanted to say, we need this and the following lines you deleted.

Reply:
no, just it has deleted.
https://github.com/sfc-aqua/quisp/pull/182/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947